dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBEncoderDecoder.java

```diff
@@ -324,6 +327,9 @@ public static Object unmarshalWithBridge(QName qname,
 
         try {
             JAXBUtils.BridgeWrapper bridge = JAXBUtils.createBridge(ctxClasses, qname, cls, anns);
+	    if (isLoggableFinest) { // Liberty change begin
+               LOG.finest("unmarshalWithBridge: bridge class: " + bridge.getClass().getCanonicalName());
```

Comment:
do we need to null check `bridge` here?

Reply:
No, should be ok here as bridge.unmarshal() is called after this trace, so it would NPE if bridge is null even without this trace.

Reply:
Will add null check here just to be safe